Description: Ignore the ruby_version as long as it's greater than 1.9
Author: Maximiliano Curia <maxy@debian.org>
Bug-Debian: http://bugs.debian.org/739128
Forwarded: no
Last-Update: 2014-04-03

--- kross-interpreters-4.12.2.orig/ruby/CMakeLists.txt
+++ kross-interpreters-4.12.2/ruby/CMakeLists.txt
@@ -1,11 +1,11 @@
 find_package(Ruby 1.9 QUIET)
 
-if( RUBY_VERSION VERSION_GREATER "1.10" )
-    message(STATUS "Found Ruby: ${RUBY_EXECUTABLE} (found version \"${RUBY_VERSION}\") but only 1.9.x supported")
-    set(RUBY_FOUND False)
-else()
+# if( RUBY_VERSION VERSION_GREATER "1.10" )
+#     message(STATUS "Found Ruby: ${RUBY_EXECUTABLE} (found version \"${RUBY_VERSION}\") but only 1.9.x supported")
+#     set(RUBY_FOUND False)
+# else()
     message(STATUS "Found Ruby: ${RUBY_EXECUTABLE} (found version \"${RUBY_VERSION}\")")
-endif()
+# endif()
 
 macro_log_feature(RUBY_FOUND
   "Ruby" "Ruby libraries" "http://www.ruby-lang.org" FALSE "1.9.x" "Needed to compile the Ruby bindings")
